app/models/respondent.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# A respondent is the person who the claimant is claiming against
+# A claim can have many respondents
+class Respondent < ApplicationRecord
+  belongs_to :address
+  belongs_to :work_address, class_name: 'Address', required: false # rubocop:disable Rails/InverseOf
```

Comment:
Same response - will get added as required but no need for it right now and not sure if there ever will be ?

Reply:
When needed, the right solution will be found - but there is no problem to find a solution for yet.  Polymorphic's seem like a good idea until you see what happens in a typical N+1 type of situation as it cannot eager load like a normal relationship, so I would only use them as required not just because we might need them.